lib/graphql_rails/errors/system_error.rb

```diff
@@ -1,14 +1,26 @@
 # frozen_string_literal: true
 
 module GraphqlRails
-  # base class which is returned in case something bad happens. Contains all error rendering tructure
+  # Base class which is returned in case something bad happens. Contains all error rendering structure
   class SystemError < ExecutionError
+    def initialize(error)
+      super(error.message)
+
+      @error = error
+    end
+
+    delegate :backtrace, to: :error
```

Comment:
Just a nit pick: `delegate` statements and other macros usually goes before `initializer`

```suggestion
    delegate :backtrace, to: :error

    def initialize(error)
      super(error.message)

      @error = error
    end
```